site/web/assets/js/classification.js

```diff
@@ -0,0 +1,140 @@
+  function updateCategory() {
+    $('select').on('change', function () {
+      var id = $(this).attr('id');
+      var getCount = id.split('-');
+      count = parseInt(getCount[1]);
+      $(this).nextAll().remove();
+      var value = $('#category-' + count + '').val();
+      if (value != "NULL") {
+        var increase = false;
+        var id = 0;
+        for (var i = 0; i < classificationDetails.length; i++) {
+          if (classificationDetails[i][3] == value) {
+            id = classificationDetails[i][1];
+            break;
+          }
+        }
+        for (var i = 0; i < classificationDetails.length; i++) {
+          if (classificationDetails[i][2] == id) {
+            if (increase == false) {
+              var category = $('#category-' + count + '').clone().appendTo('.classification-system');
+              count++;
+              category.attr('id', 'category-' + count + '').empty().append('<option value="NULL">select a category</option>');
+              $('#category-' + count + '').append('<option value="' + classificationDetails[i][3] + '">' + classificationDetails[i][3] + '</option>');
+              increase = true;
+            }
+            else {
+              $('#category-' + count + '').append('<option value="' + classificationDetails[i][3] + '">' + classificationDetails[i][3] + '</option>');
+            }
+          }
+        }
+        //include this current select element to the remove paramenter
+        updateCategory();
+      }
+    })
+  }
+
+  //remove a category to the project
+  $('.close-category').on('click', function (event) {
+    event.preventDefault();
+    if (count != 1) {
+      $('#category-' + count + '').remove();
+      count--;
+    }
+  })
+
+
+  //send ajax request for inserting classification for a project
+  $('.insert-classification').on('click', function (event) {
+    event.preventDefault();
+    var classification = '';
+    for (var j = 1; j <= count; j++) {
+      if ($('#category-' + j + '').val() != 'NULL') {
+        if (j == 1) {
+          classification = $('#category-' + j + '').val();
+        }
+        else {
+          classification = classification + '::' + $('#category-' + j + '').val();
+        }
+      }
+      else {
+        break;
+      }
+    }
+
+    if (classification != '') {
+      if ($('.update-classification').children().length <= 0) {
+        $('.update-classification').append('<p><span style="color: red">*</span> Click update project to add these classifications</p>')
+      }
+      $('.update-classification').append('<div class="categories update">\
+                        <input type="hidden" name="classification[]" value="' + classification + '" />\
+                        <span>' + classification + '</span>\
+                    <a class="remove-classification" href="#">\
+                      <i class="fa fa-close" aria-hidden="true"></i>\
+                    </a>\
+                  </div>')
```

Comment:
That's a very unusual code formatting. You can continue lines by just splitting the string.